rosdep/python.yaml

```diff
@@ -6311,6 +6311,29 @@ python3-helyos_agent_sdk-pip:
   '*':
     pip:
       packages: [helyos_agent_sdk]
+python3-hid:
+  debian:
+    pip:
+      packages: [hid]
+  fedora:
+    pip:
+      packages: [hid]
+  ubuntu:
+    pip:
+      packages: [hid]
+python3-hidapi:
+  debian: [python3-hid]
+  fedora: [python3-hidapi]
+  rhel:
+    '*': [python3-hidapi]
+    '8': null
+  ubuntu: [python3-hid]
+python3-hidapi-cffi:
+  debian: [python3-hidapi]
+  fedora:
+    pip:
+      packages: [hidapi-cffi]
```

Comment:
Let's omit the Fedora rule here for now. The package may someday become available via RPM and it may cause problems to switch the rule from pip to dnf down the road.

```suggestion
```

Reply:
OK, I removed.